db/migration/V020__insert_default_activities.sql

```diff
@@ -0,0 +1,125 @@
+INSERT INTO curriculums (principal_id, title) VALUES
+  (1, 'Professional Mentorship Program');
+
+INSERT INTO activities (title, description_text, curriculum_week, curriculum_day, start_time, end_time, duration, activity_type_id, curriculum_id) VALUES
+  -- Week 1: Day 1
+  ('Morning Standup', '', 1, 1, '10:00:00', '11:00:00', 60, 3, 1),
+  ('Self-introduction', 'Get to know each other.', 1, 1, '11:10:00', '12:00:00', 50, 1, 1),
+  -- Week 1: Day 2
+  ('Self-assessment', '', 1, 2, NULL, NULL, NULL, 2, 1),
+  ('PMP Inroduction', 'Overview of the project and technological environment.', 1, 2, '11:00:00', '12:00:00', 60, 1, 1),
+  ('Set up Local Copy of Repository', 'Clone the repository and follow the instructions to set it up on your development machine.', 1, 2, NULL, NULL, NULL, 2, 1),  
+  -- Week 1: Day 3
+  ('Morning Standup', 'Check progress.', 1, 3, '11:00:00', '12:00:00', 60, 3, 1),
+  ('Product Brief', 'Go through the Product Brief.', 1, 3, '12:10:00', '13:00:00', 50, 1, 1),
+  -- Week 1: Day 4
+  ('Game Time', '', 1, 4, '10:00:00', '10:30:00', 30, 1, 1),
+  ('Product Specification', 'Go through the Product Specification.', 1, 4, '11:00:00', '13:00:00', 120, 1, 1),
+  -- Week 1: Day 5
+  ('Self-reflection', '', 1, 5, '10:00:00', '11:00:00', 60, 4, 1),
+  ('Data flow demostration', 'Go through the existing codes on the app.', 1, 5, '11:05:00', '13:00:00', 115, 1, 1),
```

Comment:
In title case, this would be "Data Flow Demonstration", and "demonstration" is misspelled.

Reply:
I'd also like to suggest a wording change. Instead of "go through existing codes in the app", what do you think about "Review the existing code structure from the perspective of how data flows through the app."?